src/components/productForList/ProductForList.js

```diff
@@ -44,10 +42,11 @@ export const ProductForList = ({ item, token }) => {
   }, [item]);
 
   return (
-    <div className="product-container">
+    <div className={`product-container ${estimationType}`}>
       <div
         className="color-mark"
-        aria-label={`${
+        aria-label={estimationType}
+      />
           item.howSoon === 7
             ? 'soon'
             : item.howSoon === 14
```

Comment:
Problemas en el renderizado, información no valida.

Falto eliminar las líneas 50 - 55.

```suggestion
        // eliminar estas líneas
```

![image](https://user-images.githubusercontent.com/8689897/158653010-42e41e2e-48ba-4284-840d-4e84b03bd0f3.png)